superchain/superchain.go

```diff
@@ -87,7 +87,7 @@ func (a AddressList) AddressFor(contractName string) (Address, error) {
 		address = a.L1ERC721BridgeProxy
 	case "L1StandardBridgeProxy":
 		address = a.L1StandardBridgeProxy
-	case "L2OutputOrcaleProxy":
+	case "L2OutputOracleProxy":
```

Comment:
wow nice catch, surprised this didn't cause any problems before

Reply:
Yeah I guess this is a downside to this approach (of mapping runtime information to static / type information). Maybe tips the scale back a tiny bit towards using reflection...